src/test/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package eu.europeana.metis.sandbox.service.workflow;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import eu.europeana.metis.sandbox.common.exception.ServiceException;
+import eu.europeana.metis.utils.ZipFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.mock.web.MockMultipartFile;
+
+
+@ExtendWith(MockitoExtension.class)
+public class HarvestServiceImplTest {
+
```

Comment:
What about tests for OAI-PMH harvesting?

Reply:
There are still some tests missing here related to the OAI-PMH

Reply:
True. haven't figured out, how to test this here.

Reply:
True! Why is it difficult?
The harvesterOai calls can be simply mocked.

Reply:
This comment was marked as resolved..
The class has been renamed to `HttpHarvestServiceImplTest` but it's not that.
The `HarvestServiceImpl` class inside here is still missing tests for the oai functionality.
Anything that we need to know here?